service/ui_plugin_build_baron.go

```diff
@@ -49,6 +53,44 @@ func bbGetConfig(settings *evergreen.Settings) map[string]evergreen.BuildBaronPr
 	return projects
 }
 
+func bbGetTask(taskId string, execution string) (*task.Task, error) {
+	oldId := fmt.Sprintf("%v_%v", taskId, execution)
+	t, err := task.FindOneOld(task.ById(oldId))
+	if err != nil {
+		return t, err
```

Comment:
It's good practice to wrap errors with `errors.Wrap`.

Reply:
done